features/bootstrap/FeatureContext.php

```diff
@@ -8,6 +8,9 @@
 use \WP_CLI\Process;
 use \WP_CLI\Utils;
 
+use Symfony\Component\Filesystem\Filesystem,
+	Symfony\Component\Filesystem\Exception\IOExceptionInterface;
```

Comment:
We should remove the Symfony dependency, see https://github.com/wp-cli/wp-cli/pull/4209#issuecomment-314093447